worker/remoterelations/remoteapplicationworker.go

```diff
@@ -180,7 +180,7 @@ func (w *remoteApplicationWorker) loop() (err error) {
 			for i, result := range results {
 				key := change[i]
 				if err := w.relationChanged(key, result, relations); err != nil {
-					if params.IsCodeNotFound(err) {
+					if errors.IsNotFound(err) || params.IsCodeNotFound(err) {
```

Comment:
This makes me sad.

Reply:
Me too but I want to minimise the changes here to avoid unintended consequences. We need to fix all this in many, many of the api implementations.